torch/csrc/jit/codegen/cuda/ir_base_nodes.cpp

```diff
@@ -88,6 +89,22 @@ bool Val::isConstScalar() const {
   return ConstCheck::isConst(this);
 }
 
+bool Val::isZeroInt() const {
```

Comment:
Since we are getting to an extent of check for zero and check for one here, can't we have something more general like `expectInt(int value)`?

Reply:
Yeah, we will at some point. Comparison support in general needs to be built out, this would be part of it for sure. For now zero and one are special, which is why I was lazy and added these functions.